README.md

```diff
@@ -72,13 +71,14 @@ from iopipe.iopipe import IOpipe
 def handler(event, context):
   iopipe = IOpipe(CLIENT_ID)
   timestamp = time.time()
+  iopipe.start_report(timestamp, context)
```

Comment:
API wise it would seem cleaner to look more like:
`report = iopipe.Report()`
and then `report.send()`

Thoughts on this?